src/main/java/edu/kis/powp/jobs2d/drivers/adapter/LineDrawerAdapter.java

```diff
@@ -7,8 +7,12 @@
 
 public class LineDrawerAdapter extends DrawPanelController implements Job2dDriver{
 	private int startX = 0, startY = 0;
+	private DrawPanelController drawerController;
 
-	public LineDrawerAdapter() { super(); }
+	public LineDrawerAdapter(DrawPanelController drawerController_) {
```

Comment:
You should also inject line (regarding "Hardcoded line type. What about 3.3.2 test?" issue)